### DISCUSSION TITLE
Is there a way to reset keyboard shortcuts?

### DISCUSSION THREAD

**Discussion by borekb** [notaapp/talk#230] [Questions] [answered]
I've customized my keyboard shortcuts on one computer but haven't on another one. Now I'd like to try using the default keyboard shortcuts on both; what's the easiest way to do that?

**Comment by astoilkov** [comment, marked as answer]
Hmm. There isn't a very easy way to do it. You can do one of two things: 1) scroll through all shortcuts and click all reset buttons that aren't disabled, 2) edit `~/Application Support/Nota/state.json` file and leave the `config.customKeyboardShortcuts` object empty.

**Reply by borekb** [reply]
Oh, I didn't quite notice the reset button – those directly visible were all light gray because I did't customize there; only further down the list, the button was more obvious (this is not bad UX or anything, just a bit of coincidence).

Thanks, scrolling through the list and clicking the restore button a few times is the easiest 👍.

**Comment by webwesen** [comment]
adding a command palette action to reset all settings might be useful ('reset to default' or smth)
